add guides for wasm, postcss, and web workers (#2027)
snowpackjs/snowpack: www/_template/guides/tailwind-css.md

};
```

> Once the plugin are added, you can replace your native CSS `dist` imports with Tailwind's more powerful `base`, `components`, and `utilities` imports:

Reviewer: ```suggestion
Once the plugin is enabled, you can replace your native CSS `dist` imports with Tailwind's more powerful `base`, `components`, and `utilities` imports:
```
